apps/bestofjs-nextjs/src/server/api-hall-of-fame.tsx

```diff
@@ -1,32 +1,58 @@
+import uniqBy from "lodash/uniqBy";
+import mingo from "mingo";
+
 import { APIContext, FETCH_ALL_PROJECTS_URL } from "./api-utils";
 
+type FindOptions = {
+  query?: string;
+};
+
 export function createHallOfFameAPI(context: APIContext) {
   return {
-    async findMembers() {
-      const { populate, projectsBySlug } = await context.getData();
-      const { heroes } = await fetchHallOfFameData();
+    async findMembers(options?: FindOptions) {
+      const { query } = options || {};
+      const { populate, projectCollection, projectsBySlug } =
+        await context.getData();
 
-      const populateMemberProjects = (member: BestOfJS.RawHallOfFameMember) => {
-        const projects: BestOfJS.Project[] = member.projects
+      function findProjectsByOwner(owner: string) {
+        const criteria = {
+          full_name: { $regex: new RegExp("^" + owner + "/") },
+        };
+        const projects = mingo
+          .find(projectCollection, criteria)
+          .sort({ stars: -1 })
```

Comment:
-1 is sorted by ascending?

Reply:
> -1 is sorted by ascending?

-1 means descending, we want the most popular projects first.